docs/xml-config-file.rst

```diff
@@ -40,7 +41,10 @@ Types section is optional.
 Topic Types
 -----------
 
-Topic Types section allows you to specify which topic data types will be used by each participant and :ref:`types libraries`.
+Topic Types section allows you to specify which topic data types will be used by each participant
```

Comment:
What happens with topics that use dynamic types (created from the previous xml type section).

Reply:
Nothing. Dynamic types created through xml are already available to be used by the participants.